public/video-ui/src/components/FormFields/FormFieldBylinePicker.js

```diff
@@ -0,0 +1,301 @@
+import React from 'react';
+import ContentApi from '../../services/capi';
+
+export default class FormFieldBylinePicker extends React.Component {
+  state = {
+    bylineTags: null,
+    inputString: '',
+    lastAction: 'OTHER',
+    tagValue: []
+  };
+
+  componentDidMount() {
+    if (this.props.fieldValue !== this.props.placeholder) {
+      this.tagsFromString(this.props.fieldValue).then(result => {
+        this.setState({
+          tagValue: result
+        });
+      });
+    }
+  }
+
+  tagsFromString = savedTags => {
+    if (!savedTags) {
+      return new Promise(resolve => resolve([]));
+    }
+
+    return Promise.all(
+      savedTags.split('|').map(element => {
+        if (element.match('^profile/')) {
+          return ContentApi.getLivePage(element).then(capiResponse => {
+            const tag = capiResponse.response.tag;
+            return {
+              id: tag.id,
+              webTitle: tag.webTitle
+            };
+          });
+        } else {
+          return new Promise(resolve => resolve(element));
+        }
+      })
+    );
+  };
+
+  tagsToString = addedTags => {
+    return addedTags
+      .map(tag => {
+        if (typeof tag === 'string') {
+          return tag;
+        } else return tag.id;
+      })
+      .join('|');
+  };
+
+  onUpdate = newValue => {
+    this.setState({
+      tagValue: newValue
+    });
+    this.props.onUpdateField(this.tagsToString(newValue));
+  };
+
+  updateInput = e => {
+    if (this.state.lastAction === 'SPACE') {
```

Comment:
We need a way of knowing if the user is deleting or adding content, I don't think there is a way of doing that without tracking the key presses.